ui/src/main.rs

```diff
@@ -1199,71 +1198,13 @@ fn get_titlebar(cx: Scope) -> Element {
         ))
     }
 
-    cx.render(rsx!(
```

Comment:
was removing the toolbar for developer mode a mistake?

Reply:
Nah, it wasn't really useful and was more headache than it was worth just manually resizing the window.